### DISCUSSION TITLE
On Mod Deployment i get an Error

### DISCUSSION THREAD

**Discussion by ualuke** [Foxlider/FASTER#219] [Q&A]
Hey!
I followed the guide, setup my paths and mods like this:

F:\ARMA3\A3Master - A3 dedicated Server (where the exe is and where the mods should be deployed)
F:\ARMA3\A3Mods - Mod Staging folder

Profiling is activated, A3 is installed. 

I'm currently testing with two mods (CBA and Antistasi Ultimate) - these are downloaded and up to date.

But on Deployment i get this error Popup (for both mods):

![image](https://github.com/user-attachments/assets/454bc117-2c95-4503-8db3-5af74e1dfd19)

When i want to start the server, it notifies me, that two mods are missing and probably not deployed correctly. 

What could be the issue?

**Comment by Foxlider** [comment]
Seems like the server could not deploy the mod. 
Make sure the server folder is correctly set up in Deployment

**Reply by ualuke** [reply]
Thanks for the fast reply!

It should be correctly set up - when i create a new folder, and use the full path (ie "F:\ARMA3\TEST") it shows the same error.

**Reply by Foxlider** [reply]
Sounds like it can't find the path correctly... 
I'll dig deeper into it. 
In the meantime, ask on the discord for help, the community might find the answer
